packages/adapters/wagmi/src/client.ts

```diff
@@ -103,7 +103,19 @@ export class WagmiAdapter extends AdapterBlueprint {
       ...(configParams.pendingTransactionsFilter ?? {})
     }
 
-    this.createConfig({ ...configParams, networks })
+    this.wagmiChains = networks.filter(
+      caipNetwork => caipNetwork.chainNamespace === CommonConstantsUtil.CHAIN.EVM
+    ) as unknown as [BaseNetwork, ...BaseNetwork[]]
+    this.wagmiConfig = createConfig({
+      ...configParams,
+      chains: this.wagmiChains,
+      connectors: [...(configParams.connectors ?? [])],
+      transports: this.getTransports(configParams)
+    } as CreateConfigParameters)
+  }
+
+  public override construct(params: AdapterBlueprint.Params): void {
+    super.construct(params)
```

Comment:
I tried this to prevent wagmi from initializing but it's brringing new unforeseen issues